apps/meteor/app/livechat/server/hooks/markRoomRespondedAndSetVisitorAbandonment.ts

```diff
@@ -0,0 +1,62 @@
+import type { IOmnichannelRoom } from '@rocket.chat/core-typings';
```

Comment:
Do let me know if you have any concerns with this. Even if this goes as a patch, I don't feel renaming these files would affect much since we'd not really touched these callback files lately

Reply:
Yeah, I'd like to keep them separated on the "fix", and then do a refactor after merge with the other changes 👀 so if something doesnt work we know why